COMMENTS.md

```diff
@@ -0,0 +1,160 @@
+#### address_book.test.js
+
+Address Book\
+1.
```

Comment:
Why the line break after the number?

Reply:
That's how it looks in the source file but in github it's being showed as a single line